src/utils/api.jsx

```diff
@@ -1,7 +1,19 @@
 import axios from 'axios';
-import { token } from '../config/constants';
+import { Cookie } from '../services/cookie';
+import { TOKEN_URL, MAIN_URL } from '../config/constants';
+
+const token = Cookie.getToken();
+
+if (token === undefined) {
+  getAccessTokenFromAPI();
+}
 
 export const publicApiInstance = axios.create({
-  baseURL: 'https://pfa.foreca.com/api/v1',
+  baseURL: MAIN_URL,
   headers: { authorization: `Bearer ${token}` }
```

Comment:
QUESTION: is your token defined and set to headers once and for all only when app script is being bundled, isn't it?

Reply:
I have a checking for token. If where is no token I have request to generate a new one and set to cookies. From cookies I pass token to headers and make future requests.
P.S my token don't have expire time (possible to get from Foreca documentation), so I set once for all.

Reply:
Not sure where exactly are you reconfiguring headers for axios instance. Could you please point me on it?

Reply:
Yes, I am re-configuring like that (below). 
![image](https://user-images.githubusercontent.com/69632427/165497999-f934538a-cb29-4e6d-aef5-64492594cacb.png)